src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetMetadataStep.java

```diff
@@ -24,9 +27,20 @@ public CreateDatasetMetadataStep(DatasetDao datasetDao, DatasetRequestModel data
         this.datasetRequest = datasetRequest;
     }
 
+    public static Dataset setUtilityTableNames(Dataset dataset) {
+        dataset.getTables().forEach(t -> {
+            String rawDataName = FlightUtils.randomizeName(PdaoConstant.RAW_DATA_PREFIX + t.getName());
+            String sdListName = FlightUtils.randomizeName(PdaoConstant.SOFT_DELETE_PREFIX + t.getName());
+            t.rawTableName(rawDataName);
+            t.softDeleteTableName(sdListName);
+        });
+        return dataset;
+    }
```

Comment:
I'm not super happy with having this function live here, but I couldn't think of a better existing option and didn't know if it was worth making a new `Utils` class. Thoughts?

Reply:
See convo below